src/components/Main/Main.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import { Todo } from '../../types/Todo';
+import { TodoItem } from '../TodoItem';
+import { ErrorsMessages } from '../../types/ErrorsMessages';
+
+type Props = {
+  todos: Todo[],
+  handleChecker: (id: number, data: unknown) => void,
+  removeTodo: (id: number) => void,
+  isLoader: boolean,
+  loaderId: number,
+  errorMessage: (message: ErrorsMessages) => void
+};
+
+export const Main: React.FC<Props> = ({
+  todos,
+  handleChecker,
+  removeTodo,
+  isLoader,
+  loaderId,
+  errorMessage,
+}) => {
+  const isMain = todos.length > 0;
+
+  return (
+    <section className="todoapp__main">
+      {isMain && (
```

Comment:
If you always have `todos` as an array (with items or as well as an empty array) you could avoid using this checking.
If your `todos === []` then `todos.map` will do nothing. And it's a fully correct syntax for React.